evm/Cargo.toml

```diff
@@ -11,7 +11,7 @@ foundry-utils = { path = "./../utils" }
 serde_json = "1.0.67"
 serde = "1.0.130"
 hex = "0.4.3"
-ethers = { git = "https://github.com/gakonst/ethers-rs", default-features = false, features = ["solc-full"] }
+ethers = { git = "https://github.com/gakonst/ethers-rs", default-features = false, features = ["solc-full", "ws", "rustls", "ipc"] }
```

Comment:
ipc is only available on unix. 
~~and enabling this will definitely not compile on windows~~
~~so we need to feature gate this here as well~~
perhaps fine as `ipc` module should be ignored even if enabled on non linux
it should be ignored:

https://github.com/gakonst/ethers-rs/blob/367f3444ec627c6afa3bd9613644a6f903b3cb86/ethers-providers/src/transports/mod.rs#L22-L25